spec/dummy/spec/rake/assets_precompile_rake_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "rake"
+require "fileutils"
+
+require_relative "../rails_helper"
+
+describe "rake assets:precompile task" do
+  it "doesn't show deprecation message for using webpacker:clean task" do
+    ENV["SHAKAPACKER_PRECOMPILE"] = "false"
```

Comment:
I couldn't find any other way to disable shakapacker_precompile. I tried mocking `ENV[]` and `ENV.fetch` but didn't have success.

Reply:
There's a clear way to mock `ENV[]` in Ruby. please use it.

Reply:
I have done that. I mocked both `[]` and `fetch` methods, but it didn't work here. At least the tests failed locally.

I am more concerned about the cause of the current failure.

Reply:
Fixed.